packages/compass/src/main/auto-update-manager.ts

```diff
@@ -461,6 +500,55 @@ const STATE_UPDATE: Record<
       autoUpdater.quitAndInstall();
     },
   },
+  [AutoUpdateManagerState.PromptToUpdateExternally]: {
+    nextStates: [
+      AutoUpdateManagerState.UpdateDismissed,
+      AutoUpdateManagerState.ManualDownload,
+    ],
+    enter: async function (
+      updateManager,
+      _fromState,
+      updateInfo: UpdateInfo,
+      isDownloadForManualCheck: boolean
+    ) {
+      log.info(
+        mongoLogId(1_001_000_247),
+        'AutoUpdateManager',
+        'Prompting to download externally',
+        { releaseVersion: updateInfo.to, isDownloadForManualCheck }
+      );
+
+      if (!isDownloadForManualCheck) {
+        ipcMain?.broadcast('autoupdate:download-update-externally', {
+          currentVersion: updateInfo.from,
+          newVersion: updateInfo.to,
+        });
+        return;
+      }
+
+      const answer = await dialog.showMessageBox({
+        icon: COMPASS_ICON,
+        title: 'New version available',
+        message: 'A new version of Compass is available to install',
+        detail: `Compass ${updateInfo.to} is available. You are currently using ${updateInfo.from}. Would you like to download it now?`,
```

Comment:
Can we use the same text here for the new toast as well? It reads a bit nicer tbh :)

Reply:
I reached out to Misba regarding the toast content and I'd share this idea (to have same for both) with her as well.